tests/resources/policy_specification.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "",
```

Comment:
I think this was leftover from the original design.  We decided the `name` should not be part of the specification, rather it should be derived from the zone name and be its own parameter in the call to set policy.

Reply:
done